### PR TITLE
commit 8-updateReviewVotes and error handling

### PR DESCRIPTION
**Pull request by amnioc**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -245,7 +245,55 @@ describe("POST /api/reviews/:review_id/comments", () => {
...
+    const testOriginalReview = {
+      title: "Ultimate Werewolf",
+      designer: "Akihisa Okui",
+      owner: "bainesface",
+      review_img_url:
+        "https://images.pexels.com/photos/5350049/pexels-photo-5350049.jpeg?w=700&h=700",
+      review_body: "We couldn't find the werewolf!",
+      category: "social deduction",
+      created_at: new Date(1610964101251),
+      votes: 5,
```
Comment (popatre) [inline __tests__/app.test.js]: Im not sure where this gets used in this test

File: __tests__/app.test.js [hunk 2]
```diff
@@ -245,7 +245,55 @@ describe("POST /api/reviews/:review_id/comments", () => {
...
+    };
+
+    return request(app)
+      .patch("/api/reviews/3")
+      .send(testIncVotes)
+      .expect(200)
+      .then((response) => {
+        const { review } = response.body;
+        expect(review).toBeInstanceOf(Object);
+        expect(review.votes).toBe(15);
```
Comment (popatre) [inline __tests__/app.test.js:273]: We need to check you're getting the whole object back with all the keys, with the updated votes as well

File: __tests__/app.test.js [hunk 3]
```diff
@@ -245,7 +245,55 @@ describe("POST /api/reviews/:review_id/comments", () => {
...
+      });
+  });
+  it('400: should return "Invalid Data Type Used" for invalid votes data format', () => {
+    const testIncVotes = { inc_votes: "Ten" };
+    return request(app)
+      .patch("/api/reviews/4")
+      .send(testIncVotes)
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Invalid Data Type Used");
```
Comment (popatre) [inline __tests__/app.test.js:313]: Just missing a few more tests

- [ ]  400 invalid id
- [ ] 200/400 inc_vote key missing - either respond with an error or make no change.

**Review by popatre — CHANGES_REQUESTED** [review]
Looks good, just two more tests please

**Review by NRMSMITH — APPROVED** [review]
Good changes 👍 Happy for you to merge!

**Pull request merged**